CHANGELOG/CHANGELOG-1.12.md

```diff
@@ -1,4 +1,8 @@
 
+* [v1.12.2](#v1121)
+    * [Downloads for v1.12.2](#downloads-for-v1121)
+    * [KubeEdge v1.12.2 Release Notes](#kubeedge-v1121-release-notes)
+        * [Changelog since v1.12.1](#changelog-since-v1120)
```

Comment:
```suggestion
* [v1.12.2](#v1122)
    * [Downloads for v1.12.2](#downloads-for-v1122)
    * [KubeEdge v1.12.2 Release Notes](#kubeedge-v1122-release-notes)
        * [Changelog since v1.12.1](#changelog-since-v1121)
```

Reply:
fixed.